src/client/common/process/pythonProcess.ts

```diff
@@ -103,4 +110,8 @@ export class PythonExecutionService implements IPythonExecutionService {
 
         return result;
     }
+
+    public getExecutableInfo(command: string, args: string[]): IPythonExecutableInfo {
```

Comment:
Why do we need the `command` parameter?  Can't we rely on `this.getExecutablePath()` instead?

Reply:
`getExecutablePath` already relies on `getExecutableInfo`.

Reply:
Sure.  I should have said `this.pythonPath`. :)

We never pass anything except `this.pythonPath` as the first argument.  So why not drop the parameter?  `this.pythonPath` is already available in the method as a property, even on subclasses.

```suggestion
    public getExecutableInfo(args: string[]): IPythonExecutableInfo {
        return {
            command: this.pythonPath,
            args: args
        };
    }
```